docker-compose.yml

```
@@ -0,0 +1,16 @@
+version: '3'
+services:
+    backend:
+        build:
+            context: backend
+            dockerfile: Dockerfile
+        ports:
+            - 5000:5000
+    frontend:
+        build:
+            context: frontend
+            dockerfile: Dockerfile
+        depends_on:
+            - backend
+        ports:
+            - 5001:6666
```

I've updated #56 so that the frontend is using 5001, as 6666 was causing trouble.
```suggestion
            - 5001:5001
```